docs/docs/rest-api/public/api/v2/apps.raml

```diff
@@ -0,0 +1,451 @@
+    is: [ secured ]
+    responses:
+      204:
+        description: The response will contain an empty body.
+      404:
+        description: No app with this id known.
+        body:
+          application/json:
+            example: |
+              {"message":"App '/not-existing' does not exist"}
+      409:
+        description:
+          There is an already running deployment, that affects this application.
+          To override this deployment, use the force=true flag
+        body:
+          application/json:
+            example: |
+              {
+               "message":"App is locked by one or more deployments. Override with the option '?force=true'. View details at '/v2/deployments/<DEPLOYMENT_ID>'.",
+               "deployments":[{"id":"97c136bf-5a28-4821-9d94-480d9fbb01c8"}]
+              }
+
+  /restart:
+    ###
+    # Restart this application
+    #
+    post:
+      description: Restart all tasks of this application.
+      is: [ secured ]
+      responses:
+        200:
+          description:
+            A deployment is started which has a unique deployment identifier.
+            The related deployment can be fetched from the /v2/deployments endpoint.
+            If the deployement is gone from the list of deployments, than this means it is finished.
+            As long as the deployment runs, the effect of that change operation is visible only partially.
+          body:
+            application/json:
+              example: !include examples/deployments_result.json
+        404:
+          description: No task found with this `app_id`.
+          body:
+            application/json:
+              example: |
+                { "message": "App '/not_existent' does not exist" }
+
+  /tasks:
+    ###
+    # Create application
+    #
+    get:
+      description: List all running tasks for application `app_id`.
+      is: [ secured ]
+      responses:
+        200:
+          description: The list of running tasks for application `app_id`.
+          body:
+            application/json:
+              example: !include examples/app_tasks.json
+            text/plain:
+              example: |
+                minecraft_survival-world	10013	srv7.hw.ca1.mesosphere.com:31756
+        404:
+          description: No task found with this `app_id`.
+          body:
+            application/json:
+              example: |
+                { "message": "App '/not_existent' does not exist" }
+
+    ###
+    # Create application
+    #
+    delete:
+      description: Kill tasks that belong to the application `app_id`
+      is: [ secured ]
+      queryParameters:
+        host:
+          description: all tasks of that application on the supplied slave are killed
+        scale:
+          type: boolean
+          description: If `scale=true` is specified, then the application is scaled down by the number of killed tasks.
+          default: false
+      responses:
+        200:
```

Comment:
Actually, the response is different depending on the scale param:
- if `scale == true`, the result will be a deployment info
- otherwise, the result will be a list of tasks